test/end-to-end/stock/stock.assign.spec.js

```diff
@@ -128,4 +128,4 @@ function StockAssignTests() {
   });
 }
 
-describe('Stock Assign Module', StockAssignTests);
+describe.only('Stock Assign Module', StockAssignTests);
```

Comment:
This needs to be removed - it is only testing these tests.

Reply:
Sorry